community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_3/runtime/commands/expressions/IndexedInclusiveLongRange.scala

```diff
@@ -43,6 +43,7 @@ case class IndexedInclusiveLongRange(start: Long, end: Long, step: Long) extends
     }
   }
 
+  hashCode()
```

Comment:
it computes the `hashCode` at a random place in the code. It has now been removed.